ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -1,6 +1,7 @@
 # global
 from hypothesis import given, strategies as st
 
+
```

Comment:
This line should be removed as it is redudant
```suggestion
```

Reply:
sure I will